pkg/cmd/pipeline/start.go

```diff
@@ -132,6 +134,7 @@ like cat,foo.bar
 	c.Flags().StringSliceVar(&opt.ServiceAccounts, "task-serviceaccount", []string{}, "pass the service account corresponding to the task")
 	flags.AddShellCompletion(c.Flags().Lookup("task-serviceaccount"), "__kubectl_get_serviceaccount")
 	c.Flags().BoolVarP(&opt.Last, "last", "L", false, "re-run the pipeline using last pipelinerun values")
```

Comment:
Didn't we want to remove the shorthand flag for `--last` ?

Reply:
yeah we moved it to -L  https://github.com/tektoncd/cli/pull/340 and unified labels to -l as kubectl and other kube tools,

Reply:
fair enough :stuck_out_tongue: